.github/workflows/federated-integ-test.yml

```diff
@@ -0,0 +1,220 @@
+---
+
+name: Federated integ test
+
+on:  # yamllint disable-line rule:truthy
```

Comment:
With https://github.com/magma/magma/pull/12990 merged, we can actually remove the lint ignore.